packages/client/src/gen/coordinator/index.ts

```diff
@@ -2432,7 +2400,7 @@ export interface QueryMembersRequest {
    * @type {{ [key: string]: any; }}
    * @memberof QueryMembersRequest
    */
-  filter_conditions?: { [key: string]: any };
+  filter_conditions: { [key: string]: any };
```

Comment:
```suggestion
  filter_conditions?: { [key: string]: any };
```

The newest schema marks this field as optional.